src/System.Security.Cryptography.Algorithms/src/Internal/Cryptography/Helpers.cs

```diff
@@ -118,6 +118,32 @@ public static byte[] FixupKeyParity(this byte[] key)
             }
             return oddParityKey;
         }
+
+        public static void ConvertIntToByteArray(uint dwInput, ref byte[] counter)
```

Comment:
@safern The method wasn't declared public in desktop... why did it change to public here?  (Yeah, since it's in an internal type it's really internal... but why not just declare it internal?)